src/main/resources/templates/forgot_password.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta name="forgot_password" http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Streaming Platform</title>
+</head>
+<body>
+    <div>
+        <h1>Reset password</h1>
+        <p>We will be sending a reset password link to your email.</p>
+    </div>
+    <div>
+        <form method="post">
+            <p>
+                <label for="email">
+                    <input type="text" id="email" placeholder="Email" name="email" required autofocus>
+                </label>
+            </p>
+            <button type="submit">Reset</button>
+        </form>
+    </div>
+    <div>
+        <span>
+            <a th:href="@{/signup}">Sing up</a>
```

Comment:
Typo in phrase "Sign up"
```suggestion
            <a th:href="@{/signup}">Sign up</a>
```